### PR TITLE
Submitting OssAccess version 1.0.23083

### PR DESCRIPTION
**Pull request by SEA-Entwicklung**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109216&drop=dogfoodAlpha

### CODE REVIEW

**Comment by SEA-Entwicklung** [comment]
@microsoft-github-policy-service agree company="SEA Schliess-Systeme AG"

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978334972 
> 2023-06-08 05:45:10.702 [CORE] Deployment operation #0: Windows cannot install package 31916SEASchliess-Systeme-.533679AB28CA5_1.0.23083.0_x64__rts2a5cwvga10 because this package depends on a framework that could not be found. Provide the framework "Microsoft.WindowsAppRuntime.1.2" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 2000.802.31.0, along with this package to install. The frameworks with name "Microsoft.WindowsAppRuntime.1.2" currently installed are: {}
2023-06-08 05:45:10.703 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FF81F2D9A45: (caller: 00007FF81F263515) Exception(1) tid(46c) 80073CF3 Package failed updates, dependency or conflict validation.

File: manifests/s/SEA/OssAccess/1.0.23083/SEA.OssAccess.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PackageIdentifier: SEA.OssAccess
+PackageVersion: 1.0.23083
+MinimumOSVersion: 10.0.19041.0
+InstallerLocale: de-CH
+InstallerType: msix
+Scope: user
+UpgradeBehavior: install
+PackageFamilyName: 31916SEASchliess-Systeme-.533679AB28CA5_rts2a5cwvga10
+ReleaseDate: 2023-05-23
+Installers:
```
Comment (stephengillie) [inline manifests/s/SEA/OssAccess/1.0.23083/SEA.OssAccess.installer.yaml:13]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.WindowsAppRuntime.1.2
Installers:
```

**Comment by stephengillie** [comment]
With `Microsoft.WindowsAppRuntime.1.2` preinstalled, this package installs and launches normally. But the window is blank - is that expected? 
image

**Comment by SEA-Entwicklung** [comment]
> With `Microsoft.WindowsAppRuntime.1.2` preinstalled, this package installs and launches normally. But the window is blank - is that expected? image

This may happen if the Webview2 container fails to load.
What system did you use and did it come with webview2 pre-installed?

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by SEA-Entwicklung** [comment]
Since support for dependencies is implemented, can you unblock the PR?

**Pull request closed without merging**